src/model/enhance-images.test.ts

```diff
@@ -1,17 +1,10 @@
 import { Pillar } from '@guardian/types';
```

Comment:
A `ul` with more than one `li` (ie, not a Caption?) could be an edge case?

Reply:
As per my other comment - they're a thing. I think this, more than anything else, underscores why these conventions have problems and need to be replaced with more resilient solutions soon